util/dvsim/StatusPrinter.py

```diff
@@ -139,15 +139,40 @@ def exit(self):
         for target in self.status_target:
             self.status_target[target].close()
 
+class NoStatusPrinter(StatusPrinter):
```

Comment:
I feel like this should be the base "interface" class from which `StatusPrinter` itself extends from.

Reply:
OK renamed this to `StatusPrinter` and renamed it to `TtyStatusPrinter` and inherited from `StatusPrinter`.